apis/src/pwa_cache.rs

```diff
@@ -0,0 +1,27 @@
+#[cfg(feature = "ssr")]
+use actix_web::{get, HttpResponse, Responder, web};
+use walkdir::WalkDir;
+
+#[cfg(feature = "ssr")]
+#[get("/pwa-cache")]
+pub async fn pwa_cache(site_root: web::Data<String>) -> impl Responder {
```

Comment:
the right place for this file is probably apis/src/functions

Reply:
I moved it under apis/src/functions/. Could you please have a look, @PenguinWithATie?

Reply:
looks good ty